public/data/enn-dee.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "enn-dee",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/enn-dee.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/enn-dee",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/EnDee404",
+      "icon": "twitter"
+    }
+  ]
+  
```

Comment:
There's an extra unneeded line here. Might want to remove it. 🙂
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant